### PR TITLE
[tools] update schedule-migration

### PR DESCRIPTION
**Pull request by fewdan**
Signed-off-by: “fewdan” <fewdan@hotmail.com>

### What problem does this PR solve?

Problem Summary:

update schedule-migration

### What is changed and how it works?

What's Changed:

update schedule-migration
rename containerNames

### Related changes

* PR to update `chaos-mesh/website`/`chaos-mesh/website-zh`:
* Need to update Chaos Dashboard component, related issue:
* Need to cheery-pick to the release branch

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

### Release note 

```release-note
Please add a release note.
If you don't think this PR needs a release note then fill it with None.
```

### CODE REVIEW

File: tools/schedule-migration/main.go [hunk 1]
```diff
@@ -81,6 +89,38 @@ func main() {
...
+			new.Spec = append(new.Spec, yaml.MapItem{Key: "historyLimit", Value: 5})
+			new.Spec = append(new.Spec, yaml.MapItem{Key: "concurrencyPolicy", Value: "Forbid"})
+			new.Spec = append(new.Spec, yaml.MapItem{Key: "containerNames", Value: containerNames})
+			var newSpec yaml.MapSlice
+			for _, item := range old.Spec {
+				if item.Key != "scheduler" && item.Key != "containerName" {
+					newSpec = append(newSpec, item)
+				}
+			}
+			new.Spec = append(new.Spec, yaml.MapItem{Key: getKeyName(old.Kind), Value: newSpec})
```
Comment (STRRL) [inline tools/schedule-migration/main.go:113]: do not write too many duplicated codes. :cry: 

please reuse the codes in last if branch

**Review by STRRL — CHANGES_REQUESTED** [review]
rest LGTM

File: tools/schedule-migration/main.go (hunk 1 above)
Comment (fewdan) [reply tools/schedule-migration/main.go:113]: ok, i will modify it.

**Review by STRRL — APPROVED** [review]

**Review by cwen0 — APPROVED** [review]
LGTM

**Review by AsterNighT — APPROVED** [review]
LGTM

**Pull request merged**